### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by ahuth**
### Summary:

Updates
- Many dev deps
- Tailwind to the latest v2 (not v3)

### Test Plan:

- CI
- Local testing
- Chromatic

### CODE REVIEW

File: packages/components/src/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -146,6 +146,8 @@ export const CheckboxInput = React.forwardRef<HTMLInputElement, InputProps>(
   },
 );
 
+CheckboxInput.displayName = "CheckboxInput";
```
Comment (ahuth) [inline packages/components/src/Checkbox/Checkbox.tsx:149]: There's now a React lint rule enforcing providing a `displayName` where it doesn't think one will be inferred (such as when using `forwardRef`).

File: packages/components/package.json [hunk 2]
```diff
@@ -56,35 +56,35 @@
...
     "postcss-import": "^14.0.2",
     "postcss-nested": "^5.0.6",
     "react": "^16.14.0",
     "react-docgen-typescript-loader": "^3.7.2",
     "react-dom": "^16.14.0",
     "storybook-css-modules-preset": "^1.1.1",
-    "tailwindcss": "^2.2.7",
-    "ts-jest": "^27.0.7",
-    "typescript": "^4.5.4"
+    "tailwindcss": "^2.2.19",
```
Comment (anniehu4) [inline packages/components/package.json:86]: did you see issues with moving to v3? or just safer to be in a separate PR? we have this ticket but I haven't looked into the changelog much: https://app.shortcut.com/czi-edu/story/176262/upgrade-to-tailwind-3-0

**Review by anniehu4 — APPROVED** [review]

File: packages/components/package.json (hunk 2 above)
Comment (ahuth) [reply packages/components/package.json:86]: I didn't look at moving to v3. Wasn't sure if there were reasons to stay on the same major version as LP or not.

But I can't think of a reason why it would be problematic (since we ship compiled styles, not Tailwind classnames).

**Comment by ahuth** [comment]
Thanks!
